docs/examples/opentracing/README.rst

```diff
@@ -0,0 +1,97 @@
+OpenTracing Bridge Example
```

Comment:
```suggestion
OpenTracing Shim Example
```

To match the package name, and because people may interpret "shim" and "bridge" to have different meanings.